tests/basic_one_way_send.go

```diff
@@ -0,0 +1,156 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+
+	avalancheWarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	warpBackend "github.com/ava-labs/subnet-evm/warp"
+	"github.com/ava-labs/teleporter/tests/utils"
+	. "github.com/ava-labs/teleporter/tests/utils"
+
+	"github.com/ava-labs/awm-relayer/messages/teleporter"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/interfaces"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/x/warp"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	. "github.com/onsi/gomega"
+)
+
+// Ginkgo describe node that acts as a container for the teleporter e2e tests. This test suite
+// will run through the following steps in order:
+// 1. Send a transaction to the Teleporter contract on Subnet A
+// 2. Aggregate signatures and send the Warp message to Subnet B
+// 3. Verify receipt of the message on Subnet B
+func BasicOneWaySend() {
+	var (
+		teleporterMessageID *big.Int
+	)
+
+	subnetAInfo := utils.GetSubnetATestInfo()
+	subnetBInfo := utils.GetSubnetATestInfo()
```

Comment:
Typo here, you're getting subnet A info for subnet B

Reply:
Whoops, fixed now. I'm surprised the test worked with this. I wouldn't expect it to work in the general case.